README.md

```diff
@@ -1,6 +1,17 @@
-Welcome to the Adobe Stock Image community project!
+Welcome to the Adobe Stock Integration community project!
 
-# Overview
+## Overview
 
-Adobe Stock integration provides a searchable interface in the Magento Admin to explore, preview and use stock images inside PageBuilder or WYSIWYG content. Magento Admin user will be able to access Adobe stock from Media Gallery, search for images, save watermarked image preview and try it inside the content. The right fit image can be licensed through Adobe. Watermarked preview images will be replaced with licensed file in media storage and across all CMS content currently using the watermarked preview.
+Adobe Stock Integration provides a searchable interface in the Magento Admin to explore, preview and use stock images inside PageBuilder or WYSIWYG content. Magento Admin user will be able to access Adobe Stock from Media Gallery, search for images, save watermarked image preview and try it inside the content. The right fit image can be licensed through Adobe. Watermarked preview images will be replaced with licensed file in media storage and across all CMS content currently using the watermarked preview.
 
+## Documentation
+
+- Complete documentation located on the project [wiki pages](https://github.com/magento/adobe-stock-integration/wiki)   
+- Project [roadmap](https://github.com/magento/adobe-stock-integration/wiki/Adobe-Stock-Image-Integration-Roadmap) contains information about project phases and stories for each phase 
+- How to start local development described in the [installation guide](https://github.com/magento/adobe-stock-integration/wiki/Adobe-Stock-Image-Integration-Roadmap)
+
+## Community Engineering Slack
+
+To connect with Magento team and the Community, join us on the [Magento Community Engineering Slack](https://magentocommeng.slack.com). If you are interested in joining Slack, or a specific channel, send us request at [engcom@adobe.com](mailto:engcom@adobe.com) or [self signup](https://tinyurl.com/engcom-slack).
```

Comment:
I'd mention only self signup link here as it's the most straight forward and fastest way to join our slack workspace